src/CardsDriver.cpp

```diff
@@ -0,0 +1,25 @@
+
+#include "Cards.h"
+#include <iostream>
+
+
+int main(int argc, char *argv[]){
+    cout << "Try it" << endl;
+    Cards *d = new Cards("Gold", "Sell");
```

Comment:
A minor detail, but you don't really need to make this a pointer. It doesn't really make a difference, but it's fine if this is stack allocated since its running in the main loop.